lentille/src/Lentille/Client.hs

```diff
@@ -165,10 +165,10 @@ instance ToJSON IsoTime where
 data TaskData = TaskData
   { tdUpdatedAt :: IsoTime,
     tdChangeUrl :: Text,
-    tdIssueType :: [Text],
-    tdIssueId :: Text,
-    tdIssueUrl :: Text,
-    tdIssueTitle :: Text,
+    tdTtype :: [Text],
```

Comment:
It would be better to keep a user friendly name, like `tdTypes`, `tdId`, but we would have to change the toJSON implementation L178, to something like: https://github.com/softwarefactory-project/zuul-haskell/blob/ef324f772494bebef1182ee872b158bd4494bb2b/src/Zuul/Job.hs#L10-L21